lib/kuzzle/kuzzle.js

```diff
@@ -22,11 +22,11 @@
 'use strict';
 
 const path = require('path');
-const { randomBytes } = require('crypto');
 
 const { murmurHash128: murmur } = require('murmurhash-native');
 const stringify = require('json-stable-stringify');
 const Koncorde = require('koncorde');
+const nameGenerator = require('project-name-generator');
```

Comment:
Even if I really like to have human readable name I prefer if we don't add another dependency to Kuzzle for this.

We could just take the [nouns](https://github.com/aceakash/project-name-generator/blob/master/src/nouns.js) and [adjectives](https://github.com/aceakash/project-name-generator/blob/master/src/adjectives.js) and generate the name by ourselves, what do you think?

Reply:
You're no fun :expressionless: 

Ok I wrote one. And while I was at it, I created a custom list of nouns and adjectives. Collision chances are about 1 to 24'359'731'200, that should be enough for our purposes :expressionless:

Reply:
This name generator is SLOW, I don't want to spend time making it quicker for now. If we want to use it for other purposes than generating a node identifier, we might need to make it better.

Reply:
:open_mouth:

Reply:
Added a lot more names: I forgot to explore greek mythology and mystical creatures. :smile: